release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -1145,6 +1145,50 @@ With this update, the Ingress Operator no longer adds the `clientca-configmap` f
 [id="ocp-4-15-openshift-cli-bug-fixes"]
 ==== OpenShift CLI (oc)
 
+[discrete]
+[id="ocp-4-15-openshift-oc-mirror-bug-fixes"]
+==== oc-mirror
+
+Prior to this release, if a user wanted to filter operator packages by channel, that is, `mirror.operators.catalog.packages.channels`, they were required to include the default channel for the package, even if they had no intention of using bundles from that channel.  Based on this information, if the package had a default channel that was not part of the catalog, the catalog is considered invalid.
+
+[WARNING]
+====
+Modifying the default channel is very risky. Be aware that all the responsibility falls on the user. 
+====
+
+With this update, the field `defaultChannel` is introduced in the section for `mirror.operators.catalog.packages`. It allows the user to select a default `channel`. Based on this information, oc-mirror builds a new catalog, defining the selected channel in `defaultChannel` field as the package's default. 
+
+For example, if `operator A` has a catalog with four channels, and the user is interested in `stable-4.14` instead of the default channel. You can add the desired default channel manually in the given field. oc-mirror now builds a new catalog with the mentioned default channel, overriding the previous default channel. 
```

Comment:
```suggestion
For example, a catalog contains `operator A` with several channels, and `stable-4.15` as the default channel. The user is interested only in `stable-4.14` instead of the default channel. You can add the desired default channel to the imageSetConfig under the package's `defaultChannel`. oc-mirror now builds a new catalog with the defined default channel, overriding the previous default channel. 
```

Reply:
Ack.